src/Prefabs/Weapons/Axe.et

```diff
@@ -0,0 +1,76 @@
+GenericEntity : "{809488C4984AC087}Prefabs/Weapons/Handguns/M9/Handgun_M9_base.et" {
```

Comment:
Wrong base class for this?

Reply:
Might be a bit above my pay grade, I don't think I know what this should look like.  What should it look like?

Reply:
You will need to make base prefab for this that is not based on the M9 handgun most likely. You might need to start out on `Weapon_Base` or `Melee_Base`

Reply:
Man, looks like is a reasonable amount of work required to change it to that kind of base and some learning on my part. Won't have time to do it tonight my time, unfortunately. Maybe one for tomorrow unless you want to have a crack @thebonbon.